src/js/messaging/containers/Compose.jsx

```diff
@@ -98,63 +128,49 @@ export class Compose extends React.Component {
           </button>
         </div>
         <form
-            id="messaging-compose"
+            id="msg-compose"
             onSubmit={(domEvent) => { domEvent.preventDefault(); }}>
           <MessageFrom
-              cssClass="messaging-from"
+              cssClass="msg-from"
               lastName={message.sender.lastName}
               firstName={message.sender.firstName}
               middleName={message.sender.middleName}/>
           <MessageRecipient
-              errorMessage={composeMessage.errors.recipient}
-              cssClass="messaging-recipient"
+              errorMessage={validations.isValidRecipient(message.recipient) ? '' : composeMessage.errors.recipient}
```

Comment:
This is something that can be done separately from this PR, but I'm considering the idea of passing in fewer props if the components can self-contain them. For example, validation handling, classes that are likely to always be associated with the component, etc.

Specifically for this case, since `message.recipient` is already passed in the form of another prop (`recipient`), the validation can happen within the component itself.